### PR TITLE
chore: add PR template

### PR DESCRIPTION
**Pull request by spectranaut**

### CODE REVIEW

**Review by jnurthen — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--- IF EDITORIAL or CHORE, delete this template -->
+
```
Comment (spectranaut) [inline .github/pull_request_template.md:2]:
```suggestion
Closes #???

```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!--- IF EDITORIAL or CHORE, delete this template -->
```
Comment (spectranaut) [inline .github/pull_request_template.md:1]:
```suggestion
Closes #????

<!--- IF EDITORIAL or CHORE, delete the rest of this template -->
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Closes #????
+
+<!--- IF EDITORIAL or CHORE, delete the rest of this template -->
+
+Describe Change Here!
+
+# PR tracking
+
+* Related Core AAM PR:
```
Comment (scottaohara) [inline .github/pull_request_template.md:9]: other AAMs too?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+* Related AccName PR:
+* Related APG PR:
+* Any other dependent changes?
+
+# Implementation tracking
+
+* [ ] validator tests
+* WPT tests: [LINK]()
```
Comment (spectranaut) [inline .github/pull_request_template.md:17]: What are all the "testable statements" in the proposed spec?

**Pull request merged**